tests/integration/targets/mso_schema_site_anp_epg/tasks/main.yml

```diff
@@ -629,42 +584,6 @@
       template: 'Template 1'
       state: absent
 
-  - name: Ensure region for VRF1 at site level exists
-    cisco.mso.mso_schema_site_vrf_region_cidr:
-      <<: *mso_info
-      schema: '{{ mso_schema | default("ansible_test") }}'
-      template: Template 1
-      site: 'azure_{{ mso_site | default("ansible_test") }}'
-      vrf: VRF1
-      region: us-west-1
-      cidr: 10.0.0.0/16
-      state: present
-
-  - name: Ensure Private Link Label in Azure VRF subnet exist (MSO >3.3)
-    cisco.mso.mso_schema_site_vrf_region_cidr_subnet:
-      <<: *mso_info
-      schema: '{{ mso_schema | default("ansible_test") }}'
-      template: Template 1
-      site: 'azure_{{ mso_site | default("ansible_test") }}'
-      vrf: VRF1
-      region: us-west-1
-      cidr: 10.0.0.0/16
-      subnet: 10.0.0.0/24
-      private_link_label: 'New_Private_Link_Label'
-      zone: null
-
-  - name: Ensure another Private Link Label in Azure VRF subnet exist (MSO >3.3)
-    cisco.mso.mso_schema_site_vrf_region_cidr_subnet:
-      <<: *mso_info
-      schema: '{{ mso_schema | default("ansible_test") }}'
-      template: Template 1
-      site: 'azure_{{ mso_site | default("ansible_test") }}'
-      vrf: VRF1
-      region: us-west-1
-      cidr: 10.0.0.0/16
-      subnet: 10.0.1.0/26
-      private_link_label: 'PLL'
-
   - name: Add new EPG service type parameters (for version greater than 3.3)
```

Comment:
Remove all tasks related to Private Link Label (L587 to L648) as Private Link Label is a Cloud feature only

Reply:
Done